Source/Make.WarpX

```diff
@@ -99,6 +99,15 @@ ifeq ($(USE_EB),TRUE)
   USERSuffix := $(USERSuffix).EB
 endif
 
+ifeq ($(ME_SOLVER),TRUE)
+  include $(WARPX_HOME)/Source/FieldSolver/Make.package
+  CXXFLAGS += -DME_SOLVER
+  CFLAGS   += -DME_SOLVER
+  FFLAGS   += -DME_SOLVER
+  F90FLAGS += -DME_SOLVER
+  USERSuffix := $(USERSuffix).EB
+endif
+
```

Comment:
Why do we need this here? The FieldSolver/Make.package file should always be included. Do we actually need the `DME_SOLVER` flags for all those cases? I'm pretty sure the `USERSuffix := $(USERSuffix).EB` line is not needed.